EpikCord/__init__.py

```diff
@@ -1891,7 +1894,93 @@ def unload_section(self, section: Union[CommandsSection, EventsSection]):
 # class ClientGuildMember(Member):
 #     def __init__(self, client: Client,data: dict):
 #         super().__init__(data)
+class VoiceWebsocketClient:
+    def __init__(self, client:Client):
+        self.client = client
+        self.HELLO = 8
+        self.IDENTIFY = 0
+        self.HEARTBEAT = 3
+        self.sequence = None
+
+    async def connect(self, guild_id:str, channel_id:str, **kwargs):
+        self.guild_id = guild_id
+        await self.client.send_json({
+            "op": 4,
+            "d" : {
+                "guild_id": guild_id,
+                "channel_id": channel_id,
+                "self_mute": kwargs.get("self_mute", False),
+                "self_deaf" : kwargs.get("self_deaf", False)
+            }
+        })
+        def figure_out_receive(data):
+            if data["t"] == "VOICE_SERVER_UPDATE":
```

Comment:
This is so useless. Just use the `data["t"]`, it's the same thing anyways.

Reply:
:/

Reply:
im revamping the whole send request and listen system